src/gunicorn.conf.py

```diff
@@ -21,6 +21,21 @@ def post_worker_init(worker):
 log_level = "debug"
 capture_output = True
 errorlog = f"./log/error_{date}.log"
-print(f"[WGDashboard] WGDashboard w/ Gunicorn will be running on {bind}", flush=True)
-print(f"[WGDashboard] Access log file is at {accesslog}", flush=True)
-print(f"[WGDashboard] Error log file is at {errorlog}", flush=True)
+
+if os.path.exists("./ssl.ini"):
```

Comment:
Personally I am against the name "SSL" because it refers to the Secure Socket Layer protocol, which itself is deprecated. `TLS` is its successor, you probably already know. But another name would in my `PERSONAL` opinion fit better.

Reply:
hmm yeah i get your point..but is more well known haha

maybe we can change it to `ssl-tls.ini`?